_posts/2019-11-01-snowplow-r117-biskupin/index.md

```diff
@@ -0,0 +1,247 @@
+{
+	"schema": "iglu:com.snowplowanalytics.snowplow/anon_ip/jsonschema/1-0-1",
+	"data": {
+		"name": "anon_ip",
+		"vendor": "com.snowplowanalytics.snowplow",
+		"enabled": true,
+		"parameters": {
+			"anonOctets": 1,
+			"anonSegments": 1
+		}
+	}
+}
+```
+
+The IPv6 anonymization changes are backwards compatible. The `anonOctets` param will be read and useed also for IPv6 in case of missing `anonSegments`. To fully use IPv6 anonymization it's better to use the new schema (version `1-0-1`, which has `anonSegments` param). The new code can be used with `1-0-0` schema version also.
+
+## Additional event fingerprint hashing methods
+
+Credits for this contribution goes to [miike](https://github.com/miike).
+
+From this version, besides MD5, new hashing algorithms are supported:
+
+* SHA1
+* SHA256
+* SHA384
+* SHA512
+
+The schema for the configuration of the enrichment has been updated to version `1-0-1`.
+
+## Support to the spot market for core instances
+
+EmrEtlRunner now supports [EC2 spot instances][spot-instances], which can significantly reduce cost of EMR cluster by making sure the optimal instance is used.
+
+In order to enable spot instances, add a `core_instance_bid` setting to your `config.yml` file. This setting specifies a bid for an hour of EC2 spot instance in USD.
+
+```yaml
+aws:
+  emr:
+    jobflow:
+      core_instance_bid: 0.3
+```
+
+
+# Upgrading
+
+## TLS port binding and certificate
+
+A new version of the Scala Stream Collector can be found on [our Docker Hub repository](https://hub.docker.com/r/snowplow/scala-stream-collector-kinesis/tags) under `0.17.0` tag.
+
+For example to start up an ssl-enabled, auto-upgrade server, following config can be used, collector configuration should contain:
+```json
+ssl {
+  enable = true
+  redirect = true
+  port = 443
+}
+```
+However, this configuration will use environment-defined JVM-attached certificates. In order to override the default behaviour and use a custom certificate, the low-level section can be defined as (akka config section):
+```json
+ssl-config {
+  keyManager = {
+    stores = [
+      {type = "PKCS12", classpath = false, path = ${CERT_FILE}, password = "pass" }
+    ]
+  }
+}
+```
+
+## IPv6 anonymization
+
+A new version of the Snowplow Common Enrich can be found on [Maven repository](https://mvnrepository.com/artifact/com.snowplowanalytics/snowplow-common-enrich_2.11/0.38.0)
+
+The schema for the configuration of the enrichment has been updated to version `1-0-1`:
+
+```json
+{
+	"schema": "iglu:com.snowplowanalytics.snowplow/anon_ip/jsonschema/1-0-1",
+	"data": {
+		"name": "anon_ip",
+		"vendor": "com.snowplowanalytics.snowplow",
+		"enabled": true,
+		"parameters": {
+			"anonOctets": 1,
+			"anonSegments": 1
+		}
+	}
+}
+```
+
+## Additional event fingerprint hashing methods
+
+A new version of the Snowplow Common Enrich can be found on [Maven repository](https://mvnrepository.com/artifact/com.snowplowanalytics/snowplow-common-enrich_2.11/0.38.0)
```

Comment:
We don't need to duplicate this sentence for every enrichment.

Reply:
I thought it would be easier to find, but I would remove it.